### PR TITLE
NXBT-3202: Fix Configure jira site to avoid build failed

### PR DESCRIPTION
**Pull request by doulba**

### CODE REVIEW

**Comment by ataillefer** [comment]
The JIRA_PLUGIN credential refers to a [domain credential](https://github.com/nuxeo/jx-platform-jenkins/blob/master/config/credentials.yaml#L7) that is created in the platform team.

But in staging the credentials config is overridden to have [none](https://github.com/nuxeo/jx-platform-jenkins/blob/master/staging/config/credentials.yaml#L3).

So we need to test a pipeline with this configuration in staging to see how it behaves.

Maybe we can simply not define any `credentialsId: "JIRA_PLUGIN"`?


@ataillefer 

i have removed  the credentialsId

File: staging/config/unclassified.yaml [hunk 1]
```diff
@@ -1,5 +1,14 @@
 unclassified:
   gitHubPluginConfig:
   jiraGlobalConfiguration:
+    sites:
+        - credentialsId: "JIRA_PLUGIN"
```
Comment (ataillefer) [inline staging/config/unclassified.yaml]: Please always use the same indentation (2 spaces), as in https://github.com/nuxeo/jx-platform-jenkins/blob/master/config/unclassified.yaml#L8.

Comment (doulba) [reply staging/config/unclassified.yaml]: ok will try with undefined  credential

File: staging/config/unclassified.yaml [hunk 2]
```diff
@@ -1,5 +1,14 @@
 unclassified:
   gitHubPluginConfig:
   jiraGlobalConfiguration:
+    sites:
+      - credentialsId: ""
```
Comment (ataillefer) [inline staging/config/unclassified.yaml]: maybe we can even remove this whole line?

**Comment by ataillefer** [comment]
In both skaffold files, could you please apply the changes done in https://github.com/nuxeo/jx-platform-builders/commit/8c23404ede1278e2dc0fb52108d0452202a3c3e3:
- remove buildContext
- remove cache repo

File: staging/config/unclassified.yaml (hunk 2 above)
Comment (doulba) [reply staging/config/unclassified.yaml]: i will try this

```yaml
jiraGlobalConfiguration:
    sites:
      - url: "https://jira-preprod.nuxeo.com/"
```

**Review by ataillefer — APPROVED** [review]

**Review by kevinleturc — APPROVED** [review]

**Pull request merged**